authors/apps/authentication/views.py

```diff
@@ -116,19 +114,16 @@ def post(self, request):
         if not recipient:
             return Response({"message": "Enter your email"}, status=status.HTTP_400_BAD_REQUEST)
         # generate a new token for each email entered
-        token = jwt.encode({"email": recipient},
```

Comment:
pass this token as a param in the url. That is, the final url should be
`
https://ah-fulldeck-frontend-staging.herokuapp.com?token=some-long-token
`
the token might be required during reset of password